lnrpc/rpc.proto

```diff
@@ -658,6 +658,11 @@ message Transaction {
     repeated string dest_addresses = 8 [ json_name = "dest_addresses" ];
 }
 message GetTransactionsRequest {
```

Comment:
Ideally we'd also like to add start and end time times like we do with [forwarding events](https://github.com/lightningnetwork/lnd/blob/master/channeldb/forwarding_log.go#L156) within the switch.

Reply:
Exactly

Reply:
Exactly. But I'd like to modify this feature next PR.